homeassistant/components/zwave_js/api.py

```diff
@@ -542,7 +693,18 @@ def device_registered(device: DeviceEntry) -> None:
         ),
     ]
 
-    result = await controller.async_begin_inclusion(inclusion_strategy)
+    try:
+        result = await controller.async_begin_inclusion(
```

Comment:
What can raise a `ValueError` here ?

Reply:
1. When provisioning info is provided for a non S2 inclusion strategy
2. When the input QR code string is too short or doesn't start with the right prefix
3. When a SmartStart QR code is used as the input - we pre-provision the node in the controller but raise an error so the user knows something different than what they expected happened

Reply:
now we don't pre-provision the node in the controller in case 3, we just throw an exception